server/models.py

```diff
@@ -145,6 +145,13 @@ def icao_must_exist(cls, v) -> str|None:
 
         return v
 
+class AirlineModel(CustomModel):
+    id:   int
```

Comment:
Do we need an id column? I think the `icao` is sufficient since that's its purpose for airlines and it should (is?) unique.